Automattic/pocket-casts-ios
# Halloween icon: add 152x152 iPad mini icon

After submitting the latest beta we received a message that the iPad mini 152 x 152 icon is missing. This PR adds it.

## To test

1. Run the app on iPad mini
2. Login into a Plus account
3. Change the icon to Halloween
4. ✅ Ensure the icon changes

To test if the correct icon is being displayed you can change the `AppIcon-Halloween-152@2x.png` to whatever you want and perform the steps above, your image should show as the app icon on iPad mini.

## Checklist
